packages/flutter_tools/lib/src/project.dart

```diff
@@ -153,7 +153,7 @@ class IosProject {
   /// The parent of this project.
   final FlutterProject parent;
 
-  static final RegExp _productBundleIdPattern = RegExp(r'^\s*PRODUCT_BUNDLE_IDENTIFIER\s*=\s*(.*);\s*$');
+  static final RegExp _productBundleIdPattern = RegExp(r'^\s*PRODUCT_BUNDLE_IDENTIFIER\s*=\s*["]?(.*?)["]?;\s*$');
```

Comment:
You can get away with just `"?` instead of `["]?` in your optional quote matching:
```
r'^\s*PRODUCT_BUNDLE_IDENTIFIER\s*=\s*"?(.*?)"?;\s*$'
```

Reply:
As already written in the previous pull-request #22851, this is just my personal favor and has no impact in the regex at all